docs/content/en/docs/reference/crd-reference/analysisdefinition.md

```diff
@@ -132,21 +139,21 @@ spec:
       target:
         failure:
           <operator>:
-            fixedValue: integer> |
+            fixedValue: <integer-quantity> |
             inRange: | notInRange:
               lowBound: <integer-quantity>
               highBound: <integer-quantity>
         warning:
           <operator>:
-            fixedValue: integer> |
+            fixedValue: <integer-quantity> |
```

Comment:
i don't think this has to be an integer, it can be anything that the quantity object allows

Reply:
Or maybe we need to explain what an `integer-quantity` is?

Reply:
i'm also fine with just using quantity - I thought `integer-quantity` might illustrate that this can be either a plain integer or something following the quantity notation, but given that the quantity notation also includes normal integers it will suffice to just use `quantity`

Reply:
I could note it as <integer> | <quantity> to show that either can be used.  Or maybe add to the description of quanity that it can represent either an integer or a number that would have a decimal point...

Reply:
@mowies @bacherfl Could you look at the rendered version of this page.  Part of the mess you guys were seeing was because the "Example" was written as part synopsis, part example -- bad writer!  I think the current synopsis is fairly clear and it does show that either integer or quantity is a valid value for some fields.  See what you think.

Reply:
Ok this looks much better, nice!